### DISCUSSION TITLE
Funktion nur zwei Tage? Refresh Token?

### DISCUSSION THREAD

**Discussion by 290989** [flobz/psa_car_controller#878] [General]
Hi zusammen,

ich habe gesehen, dass es scheinbar etwas neues mit einem Refresh Token etc. gibt?
Ich bekomme nach dem neuen Vorgehen (mit dem Code über den Editor) alles zum laufen.
Das Ganze läuft dann jedoch nur so 1-2 Tage, danach verbindet sich PSACC nicht mehr.
Dann muss Ich die Anmeldung/Verifizierung wieder neu machen.

Bei mir läuft PSACC als AddOn in HomeAssistant.

Gibt es dafür schon eine Lösung? Hat es mit dem Refresh Token zu tun?

Vielen Dank,
Gruß Daniel

**Comment by jmeister79** [comment]
Same issue here

**Reply by gerar19** [reply]
same here
